understanding/22/target-size-minimum.html

```diff
@@ -104,15 +109,16 @@ <h3>Spacing</h3>
 
             <figure id="target-text-buttons-two-rows">
                 <img src="img/target-text-buttons-two-rows.svg" width="785" height="105" alt="Two rows of buttons only 16 CSS pixels high, with no spacing between them. The buttons are overlaid with the 24 CSS pixel diameter circles, and all the circles either overlap another circle, or another target." />
-                <figcaption>Two rows of buttons, both with a height of only 16 CSS pixels. The rows are close, with only a 1 CSS pixel gap between them, which means that the 24 CSS pixel spacing circles of the targets in one row will intersect the targets (and their circles, depending on their respective widths) in the other line, thus failing the Success Criterion (image shown to scale - <a href="img/target-text-buttons-two-rows.svg">see the scalable original version</a>)</figcaption>
+                <figcaption>Two rows of buttons, both with a height of only 16 CSS pixels. The rows are close, with only a 1 CSS pixel gap between them. This means that the 24 CSS pixel spacing circles of the targets in one row will <em>intersect</em> the targets (and their circles, depending on their respective widths) in the other line, thus failing the Success Criterion. Image shown to scale - <a href="img/target-text-buttons-two-rows.svg">see the scalable original version</a>.</figcaption>
             </figure>
-            <p>The following two illustrations show how menu items can be adjusted to properly meet this requirement. In the first illustration, the <q>About us</q> menu has been activated, showing that each of the menu item targets (text and padding) has a 24 CSS pixel height. In the second illustration, the same menu is expanded, but the menu items only achieve 18 CSS pixels in height.</p>
+            <p>The following two illustrations show how menu items can be adjusted to properly meet this requirement. In the first illustration, the <q>About us</q> menu has been activated, showing that each of the menu item targets has a 24 CSS pixel height, and so passes. In the second illustration, the same menu is expanded, but the menu items only achieve 18 CSS pixels in height, and so fails.</p>
```

Comment:
We have a number disagreement. It is the menu items that are being judged, not the menu.
```suggestion
            <p>The following two illustrations show how menu items can be adjusted to properly meet this requirement. In the first illustration, the <q>About us</q> menu has been activated, showing that each of the menu item targets has a 24 CSS pixel height, and so pass. In the second illustration, the same menu is expanded, but the menu items only achieve 18 CSS pixels in height, and so fail.</p>
```